pages/question/question.css

```diff
@@ -24,6 +24,7 @@
   justify-content: space-around;
   gap: 2rem;
   margin-top: 4rem;
+  margin-bottom: 4rem;
```

Comment:
Just a small suggestion, maybe you can use the shorthand property for margin or you can use `margin-block` property for setting margins on the y-axis.

Reply:
Sure